targets/TARGET_NXP/TARGET_MCUXpresso_MCUS/TARGET_LPC55S69/TARGET_M33_NS/device/TOOLCHAIN_IAR/LPC55S69_cm33_core0_flash.icf

```diff
@@ -59,7 +59,7 @@ if (!isdefinedsymbol(MBED_BOOT_STACK_SIZE)) {
 }
 
 define symbol __stack_size__ = MBED_BOOT_STACK_SIZE;
-define symbol __heap_size__  = 0x4000;
+define symbol __heap_size__  = 0x10000;
```

Comment:
instead of this one, we should keep as it was and change to dynamic stack

`define block HEAP       with expanding size, alignment = 8, minimum size = __heap_size__  { };`

Reply:
@0xc0170 . Just a side note - shouldn't we migrate all targets to use dynamic heap allocation with IAR? Do you know if anyone is working on this? If not then who could take care of this? 
This could help us avoid a lot of issues of this kind.

Reply:
We should, but it's a lot of work :/

Reply:
Done

Reply:
@MarceloSalazar Can we bring this up again to partners to get this update to every target?